src/lib/swapper/swappers/utils/helpers/helpers.test.ts

```diff
@@ -34,3 +41,26 @@ describe('normalizeIntegerAmount', () => {
     expect(result4).toEqual('586084736227728')
   })
 })
+
+describe('getTreasuryAddressForReceiveAsset', () => {
```

Comment:
Ditto, if this is a mapping of `ChainId` to address, all we'd need to do is ensure `DAO_TREASURY_BY_CHAIN_ID` is kept up-to-date, no need to test the actual getters, as long as the property is there, we're good. We could keep the `expect().toThrow()` test

Reply:
Actioned 👌 https://github.com/shapeshift/web/commit/0348c29fd253d00f8152957a948d872a9941a8cf